optuna/samplers/nsgaii/_sampler.py

```diff
@@ -104,9 +103,23 @@ class NSGAIISampler(BaseSampler):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.5.0.
 
+        elite_population_selection_strategy:
+            The selection strategy for determining the individuals to survive from the current
+            population pool. Default to :obj:`None`.
+
+        child_generation_strategy:
+            The strategy for generating child parameters from parent trials. Defaults to
+            :obj:`None`.
+
         after_trial_strategy:
-            A process to be conducted after each trial. Defaults to
-            :class:`~optuna.samplers.nsgaii._after_trial_strategy.NSGAIIAfterTrialStrategy`.
+            A set of procedure to be conducted after each trial. Defaults to :obj:`None`.
+
+            .. note::
+
+                The arguments ``elite_population_selection_strategy``,
+                ``child_generation_strategy``, and ``after_trial_strategy`` was added in v3.3.0 as
+                an experimental feature. The interface may change in newer versions without prior
+                notice. See https://github.com/optuna/optuna/releases/tag/v3.3.0.
```

Comment:
It seems the note has been added to only one argument, specifically `after_trial_strategy`, and not to all three. While it might seem redundant, would it be worth considering adding similar notes to `elite_population_selection_strategy` and `child_generation_strategy` as well?

<img width="1072" alt="image" src="https://github.com/optuna/optuna/assets/3255979/3803bd15-91f4-4c2b-872b-b23b076f42a2">